commands/tweet.js

```diff
@@ -0,0 +1,33 @@
+var TwitterClient = require('../libs/twitter-client-instance');
+
+function getLatestStatus(tweets) {
+    return tweets.statuses[0];
+}
+
+module.exports = function (bb8) {
+    console.log('Let\'s Get Tweets!!');
+
+    var executeTwitterCommand = function () {
+        TwitterClient.get('search/tweets', {q: 'bb8bbc'}, function (error, tweets) {
```

Comment:
maybe pass the query param through from the CLI maybe with something like --hashtag="bb8bbc" so we aren't hardcoding it so it can be anything :smile: